Pinta.Core/DialogResponses.cs

```diff
@@ -0,0 +1,7 @@
+namespace Pinta.Core;
+
+public enum DialogResponses
+{
+	OK,
+	Bug,
```

Comment:
The "Bug" response type is pretty specific to the error dialog, so perhaps this enum should be just for the possible error dialog responses rather than being generic?

Having a standard place for common response strings and their labels is good, though - this is just more about when the enum is returned to some calling code

Reply:
I have added a new `Messages` folder in the `Pinta.Core` project. "Messages" in the programming sense, not in the UI (i.e. message dialogs) sense.